src/Microsoft.CodeAnalysis.Razor/Microsoft.CodeAnalysis.Razor.csproj

```diff
@@ -10,8 +10,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.Common" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="$(MicrosoftCodeAnalysisCommonPackageVersion)" />
```

Comment:
These package is also used at runtime. Why are we not utilizing the lineup versions to ensure MVC + this are consistent?

Reply:
These versions should still be consistent. The automation that sets versions in build/dependencies.props uses Internal.AspNetCore.Universe.Lineup to ensure we are consistent across repos.